include/main_functions_home.h

```diff
@@ -51,7 +54,6 @@ int change_mode(char* arg);
  *		- dict_name: string indicating dictionary file name
  * return: void
  */
-void main_page(bool* quit, int *mode, char* file_name, char* dict_name);
-
+void main_page(bool *quit, int *mode, char *filename, char *dict, bool *color);
```

Comment:
Why is this a `bool *` and not just a `bool`?

Reply:
`*quit` and `*color` are booleans called here because the main menu can change their value, and if I just called the value, I wouldn't be able to change it to affect future prints to output. main_page is one of a few functions that can change that value.
I've changed the function calls for functions who won't affect the value, such as the many you commented on for shellstrings, but I think for main_page it might be necessary to do `*quit` and `*color`.